assets/src/routes/routes.js

```diff
@@ -27,8 +27,8 @@ const routes = (courseId, activeViews, activeOnly = true) => {
       path: `/courses/${courseId}/assignments`,
       title: 'Assignment Planning',
       icon: Grade,
-      description: 'See what assignments have the greatest impact on your grade.',
-      image: '/static/images/assignments_planning_icon.png',
+      description: 'See what assignments have the greatest impact on your grade and set grade goals.',
+      image: '/static/images/assignment_planning_icon.png',
```

Comment:
Nit-pick: the naming could be clearer here. The single character difference between the other is easy to miss. We could say something like `assignment_planning_goals.png` to avoid the `v1`, `v2, `old`, `new` language.

Reply:
I figured the original one was going away so it wouldn't be confusing for long :-p